modules/nested-tabs-html/widgets/nested-tabs-html.php

```diff
@@ -1123,24 +1122,41 @@ protected function render_tab_titles_html( $item_settings ): string {
 
 	protected function render_tab_containers_html( $item_settings ): string {
 		ob_start();
-		$this->print_child( $item_settings['index'] );
+		$this->print_child( $item_settings['index'], $item_settings );
 		$container_html = ob_get_clean();
 
-		$setting_key = $this->get_repeater_setting_key( 'tab_content', 'tabs', $item_settings['index'] );
-		$attribute_selector = '" data-id=';
-		$active_class = 0 === $item_settings['index'] ? ' e-active"' : '"';
+		$attribute_selector = 'class="';
+		$active_class = 0 === $item_settings['index'] ? $attribute_selector . 'e-active ' : $attribute_selector;
 
-		$this->add_render_attribute( $setting_key, [
-			'id' => $item_settings['container_id'],
-			'role' => 'tabpanel',
-			'aria-labelledby' => $item_settings['tab_id'],
-			'data-tab-index' => $item_settings['tab_count'],
-			'style' => '--n-tabs-title-order: ' . $item_settings['tab_count'] . ';',
-		] );
+		return str_replace( $attribute_selector, $active_class, $container_html );
+	}
 
-		$render_attributes = $active_class . $this->get_render_attribute_string( $setting_key ) . str_replace( '"', '', $attribute_selector );
 
-		return str_replace( $attribute_selector, $render_attributes, $container_html );
+	/**
+	 * Print the content area.
+	 *
+	 * @param int $index
+	 * @param array $item_settings
+	 */
+	public function print_child( $index, $item_settings = [] ) {
```

Comment:
I have copied the existing approach from the Mega Menu. This approach is better than my previous approach.